package.json

```diff
@@ -12,21 +12,21 @@
   "repository": "https://github.com/single-spa/vue-cli-plugin-single-spa",
   "bugs": "https://github.com/single-spa/vue-cli-plugin-single-spa/issues",
   "dependencies": {
-    "semver": "^7.3.2",
-    "standalone-single-spa-webpack-plugin": "^1.2.0",
-    "systemjs-webpack-interop": "^2.3.1"
+    "semver": "^7.3.5",
+    "standalone-single-spa-webpack-plugin": "^2.0.2",
+    "systemjs-webpack-interop": "^2.3.7"
   },
   "husky": {
     "hooks": {
       "pre-commit": "pretty-quick --staged && eslint ."
     }
   },
   "devDependencies": {
-    "eslint": "^7.7.0",
+    "eslint": "^7.28.0",
```

Comment:
Changing the package.json dependency versions should have impacted the pnpm-lock.yaml file, but I do not see changes to the pnpm-lock.yaml file. Please include those changes in this PR, or revert the dependency upgrades.